File: class14/cpp/AP1401-AprovadoReprovado.cpp
#include <iostream>
int main() {
    double media;
    std::cout << "Digite a media do aluno: ";
    std::cin >> media;
    if (media >= 7) std::cout << "APROVADO" << std::endl;
    else std::cout << "REPROVADO" << std::endl;
    return 0;
}

File: class14/java/AP1401-AprovadoReprovado.java
import java.util.Scanner;

public class AprovadoReprovado {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Digite a media do aluno: ");
        double media = sc.nextDouble();
        if (media >= 7) System.out.println("APROVADO");
        else System.out.println("REPROVADO");
    }
}
